tree/tree/src/TChain.cxx

```diff
@@ -268,87 +268,111 @@ Int_t TChain::Add(TChain* chain)
-///    is called and entry refers to an entry in the 3rd file, for example,
-///    this forces the Tree headers in the first and second file
-///    to be read to find the number of entries in these files.
-///    Note that if one calls TChain::GetEntriesFast() after having created
-///    a chain with this default, GetEntriesFast will return TTree::kMaxEntries!
-///    TChain::GetEntries will force of the Tree headers in the chain to be
-///    read to read the number of entries in each Tree.
-///
-/// D. The TChain data structure
-///    Each TChainElement has a name equal to the tree name of this TChain
-///    and a title equal to the file name. So, to loop over the
-///    TFiles that have been added to this chain:
+/// Note the following:
+/// - The "query" token is to be interpreted by the remote server.
+/// - Wildcards may be supported in URLs, depending on the protocol plugin and
+///   the remote server.
+/// - \p http or \p https URLs can contain a query identifier without
+///   \p tree_name, but generally URLs can not be written with them because of
+///   ambiguity with the wildcard character. (Also see the documentation for
+///   TChain::AddFile, which does not support wildcards but allows the URL name
+///   to contain a query).
+/// - The rules for tagging the name of the tree in the file are the same as
+///   in the format above.
+///
+/// \paragraph nentries_par Details of the nentries parameter
+/// Depending on the value of the parameter, the number of entries in the file
+/// is retrieved differently:
+/// - If <tt>nentries <= 0</tt>, the file is connected and the tree header read
+///   in memory to get the number of entries.
+/// - If <tt>nentries > 0</tt>, the file is not connected, \p nentries is
+///   assumed to be the number of entries in the file. In this case, no check is
+///   made that the file exists and that the corresponding tree exists as well.
+///   This second mode is interesting in case the number of entries in the file
+///   is already stored in a run data base for example.
+/// - If <tt>nentries == TTree::kMaxEntries</tt> (default), the file is not
+///   connected. The number of entries in each file will be read only when the
+///   file will need to be connected to read an entry. This option is the
+///   default and very efficient if one processes the chain sequentially. Note
+///   that in case TChain::GetEntry(entry) is called and entry refers to an
+///   entry in the 3rd file, for example, this forces the tree headers in the
+///   first and second file to be read to find the number of entries in these
+///   files. Note that calling TChain::GetEntriesFast after having
+///   created a chain with this default returns TTree::kMaxEntries ! Using
+///   TChain::GetEntries instead will force all the tree headers in the chain to
+///   be read to get the number of entries in each tree.
+///
+/// \paragraph datastructure The TChain data structure
```

Comment:
```suggestion
/// \paragraph The TChain data structure
```

Reply:
Unfortunately the \paragraph command wants a name for the paragraph itself https://www.doxygen.nl/manual/commands.html#cmdparagraph . I was actually just looking for some section highlighting, maybe using this command is a bit overkill

Reply:
ah i see :+1: